### ISSUE TITLE
Await not recognised in Coffeescript

### ISSUE THREAD

**Issue by DinisCruz** [wallabyjs/public#1625]
### Issue description or question

Hi, the await method doesn't seem to be recognised inside the wallaby execution environment 

![image](https://user-images.githubusercontent.com/656739/39087065-f91c3764-4592-11e8-99cd-ff399f76c3a1.png)

it works in mocha

![image](https://user-images.githubusercontent.com/656739/39087076-19836e64-4593-11e8-9b97-8d8ec7ca4515.png)

Any ideas?

### Wallaby.js configuration file

```javascript
module.exports = function (wallaby) {
    return {
        files: [
            'src-node/src/**/*.coffee'        ],

        tests: [
            'src-node/test/**/*.coffee'
        ],
        compilers: {
            '**/*.js?(x)': wallaby.compilers.babel({}),
        },
        env: {
            type: 'node',
            runner: 'node'
        }
    }
}
```

### Code editor or IDE name and version

Visual Studio Code v1.?
WebStorm v?
IntelliJ IDEA v?
Atom v1.?
Visual Studio v?
Sublime Text v3 build ?

### OS name and version

Windows
OSX
Linux

**Comment by DinisCruz** [comment]
ok, so running your sample gives me the same error

![image](https://user-images.githubusercontent.com/656739/39093744-672c9184-461c-11e8-9ada-8af75ce88023.png)

**Comment by DinisCruz** [comment]
`console.log process.version` returns `v9.10.1 `

**Comment by DinisCruz** [comment]
Humm, I just checked my wallaby license and it as expired almost a year ago

![image](https://user-images.githubusercontent.com/656739/39093785-f2229860-461c-11e8-85cc-d21d55c442a3.png)

could that be the prob?

**Comment by ArtemGovorov** [comment]
> Humm, I just checked my wallaby license and it as expired almost a year ago

Was just going to ask your wallaby core version (printed in the wallaby console), but I think it is outdated (because of the expired license) and most likely is the issue. 

Just saw you've updated your license - please enter it, restart WS and start wallaby - it should update, restart wallaby again (so that it displayed core v.1.0.584 in Wallaby Console) and let me know the result.

**Comment by DinisCruz** [comment]
btw, where can I see the new features since last July :)

**Comment by ArtemGovorov** [comment]
You may [start here](https://github.com/wallabyjs/public/blob/master/CHANGELOG.md#changes-and-bug-fixes-2) and scroll up.

**Comment by DinisCruz** [comment]
for reference I just had this issue again and it looks like when the wallaby.js config file is not on the same location as the node_modules folder, the compilation doesn't work

![image](https://user-images.githubusercontent.com/656739/39842580-d7e37210-53de-11e8-9519-ee38c11e59d1.png)

when I moved that wallaby-async.js file to the parent folder, it worked ok

**Comment by ArtemGovorov** [comment]
Yes, that is the current limitation of wallaby CoffeeScript compiler integration. Also for a few other reasons we recommend to keep wallaby config in the project's root (where the `node_modules` are) anyway.
